README.rst

```diff
@@ -1,6 +1,11 @@
 Cellarium Cloud
 ===============
 
+..
+    This section is kind of awkward but I don't know how to fix it.  I feel like it should probably describe what
+    Cellarium Cloud actually does, like the way its intended to be used by users.  Something about cell annotation
+    and related stuff, I guess?
+
 **What's Cellarium Cloud?** Cellarium Cloud is the backend application that serves the Cellarium Cloud platform.
 This is a backend application that provides API for Cellarium Client tool. It is a mono-repository that contains all
 the services that are required to run the Cellarium Cloud platform.
```

Comment:
Maybe something like:

Cellarium Cloud contains the service code that backs supports Cellarium Cloud platform by providing APIs for the Cellarium Client tool. The APIs can be access with direct REST access or, preferably, via the cellarium-cas Python client.

Reply:
Ooo I like that better.  Thank you!